scrit-whitepaper.md

```diff
@@ -216,7 +216,7 @@ following (see Figure 1):
 
 -   Start of the signing epoch, which refers to start of the key
     rotation epoch and must be globally coordinated between mints.
--   Input DBCs: List of unblinded DBC messages, not including mint
+-   Input DBCs: List of  unblinded DBC messages, not including mint
```

Comment:
Question here because it's not clear to me at this stage: were the input DBCs blinded and then un-blinded to make it to the tx, or were they never blinded?

Reply:
It is both cases. Each Input DBC is either a never blind DBC, or a user-unblinded, previously blinded DBC.